src/SpeedManager.cc

```diff
@@ -7,7 +7,7 @@ namespace openmsx {
 SpeedManager::SpeedManager(CommandController& commandController)
 	: speedSetting(commandController, "speed",
 	       "controls the emulation speed: higher is faster, 100 is normal",
-	       100, 1, 10000, Setting::Save::NO)
+	       100.0, 0.01, 10000.0, Setting::Save::NO)
```

Comment:
For consistency, also change the constants (e.g. add `.0`) in `fastforwardSpeedSetting`.

Reply:
Done in the latest version.